tests/test_image_property_issue_manager.py

```diff
@@ -84,7 +81,7 @@ def test_set_params(self, params, expected_params, issue_manager):
         ],
     )
     def test_get_defer_set(
-        self, issue_types, imagelab_info, expected_defer_set, issue_manager
+        self, issue_types, agg_computations, expected_defer_set, issue_manager
     ):
         """Tests image_property_issue_manager._get_defer_set(). Cases covered:
         1. If two issue types use the same image property, skip one
```

Comment:
Can you update docstring with description of `agg_computations` ?  Hard to understand what that is exactly otherwise